.gitpod.yml

```diff
@@ -22,7 +22,7 @@ ports:
     visibility: private
   # Direct-connect ddev-webserver port that is the main port
   - port: 8080
-    onOpen: open-preview
```

Comment:
Hmm... 8080 is the primary use case here. If it's not working we have trouble. It's the only one that really should *not* be ignored.

Reply:
@rfay 8080 just need a bit more time. When ddev makes it available, the site cannot be shown yet. But with have `gp await port` command that works well. This is all for displaying 8080 preview automatically. The user can always open it manually.

Reply:
@rfay in the last commit, I restored the port 8080 announcement to the user